chromadb/segment/impl/vector/local_persistent_hnsw.py

```diff
@@ -220,6 +225,20 @@ def _apply_batch(self, batch: Batch) -> None:
         if self._num_log_records_since_last_persist >= self._sync_threshold:
             self._persist()
 
+            with self._db.tx() as cur:
```

Comment:
nice :)

Reply:
Only thing left here, I think is to read the max_seq_id from sqlite upon init as opposed to from the pickle (L115)

Reply:
28bf0455